docs/hub-deployment-guide/configure-auth/dummy-auth.md

```diff
@@ -0,0 +1,110 @@
+# Dummy Authentication
+
+This documentation covers setting up the [Dummy Authenticator](https://jupyterhub.readthedocs.io/en/stable/reference/authenticators.html#the-dummy-authenticator)
+for a hub.
+
+For this section, it may be useful to set the following environment variables in your terminal.
+
+```bash
+export CLUSTER_NAME=cluster_name
+export HUB_NAME=hub_name
+```
+
+## Use a specific branch of the homepage repository
+
+The Dummy Authenticator requires presenting the user with a username and password input field, rather than the typical "log in" button.
+This means we cannot use the default homepage template since this will not provide these input fields.
+Instead we use a specialised branch of the homepage repo which allows us to have the username and password input fields, along with providing specific info about and for each community: [`username-and-password-homepage`](https://github.com/2i2c-org/default-hub-homepage/tree/username-and-password-homepage).
+
+In the `${HUB_NAME}.values.yaml` file, include the following config.
+
+```yaml
+jupyterhub:
+  custom:
+    homepage:
+      gitRepoBranch: "username-and-password-homepage"
+      templateVars: [...]  # These values are as normal
+```
+
+## (Recommended) Using Dummy Authenticator with a global password
+
+We strongly recommend using a global password with the authenticator.
```

Comment:
In our infrastructure, we should *only* support this with a global password. For fully password-less access, there is documentation under 'ephemeral hub' that uses tmpauthenticator i think.